.gitignore

```diff
@@ -57,6 +57,7 @@ local_settings.py
 # Flask stuff:
 instance/
 .webassets-cache
+flask_session
 
 # Scrapy stuff:
 .scrapy
```

Comment:
what is scrapy used for?

Reply:
The scrapy stuff (and the rest of the gitignore) was already like that in the original master branch; the only line I added was for the flask_session folder.